demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -0,0 +1,289 @@
+ limitations under the License.
+"""
+
+import logging as log
+import sys
+from argparse import ArgumentParser, SUPPRESS
+from pathlib import Path
+from time import perf_counter
+
+import cv2
+import numpy as np
+
+sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python'))
+sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python/openvino/model_zoo'))
+
+from model_api.models import OutputTransform, get_instance_segmentation_model
+from model_api.performance_metrics import PerformanceMetrics
+from model_api.pipelines import get_user_config, AsyncPipeline
+from model_api.adapters import create_core, OpenvinoAdapter, RemoteAdapter
+
+import monitors
+from images_capture import open_images_capture
+from helpers import resolution, log_latency_per_stage
+
+log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.DEBUG, stream=sys.stdout)
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument('-m', '--model', help='Required. Path to an .xml file with a trained model.',
+                      required=True)
+    args.add_argument('--adapter', help='Optional. Specify the model adapter. Default is openvino.',
+                      default='openvino', type=str, choices=('openvino', 'remote'))
+    args.add_argument('-i', '--input', required=True,
+                      help='Required. An input to process. The input must be a single image, '
+                           'a folder of images, video file or camera id.')
+    args.add_argument('-d', '--device', default='CPU', type=str,
+                      help='Optional. Specify the target device to infer on; CPU, GPU, HDDL or MYRIAD is '
+                           'acceptable. The demo will look for a suitable plugin for device specified. '
+                           'Default value is CPU.')
+    args.add_argument('-pt', '--prob_threshold',
+                      help='Optional. Probability threshold for detections filtering.',
+                      default=0.5, type=float, metavar='"<num>"')
+    args.add_argument('--keep_aspect_ratio',
+                      help='Optional. Force image resize to keep aspect ratio.',
+                      action='store_true')
+    args.add_argument('--labels', help='Optional. Labels mapping file.', default=None, type=str)
+    args.add_argument('--target_bgr', default=None, type=str,
+                      help='Optional. Background onto which to composite the output (by default to green field).')
+    args.add_argument('--blur_bgr', default=False, action='store_true',
+                      help='Optional. Blur background.')
+
+    infer_args = parser.add_argument_group('Inference options')
+    infer_args.add_argument('-nireq', '--num_infer_requests', help='Optional. Number of infer requests.',
```

Comment:
`nireq`, `nthreads`, `nstreams` aren't used, are they?

Reply:
In the script are used long names of these arguments (for example `num_infer_requests`) like in other demos.